merlin/models/tf/inputs/embedding.py

```diff
@@ -524,7 +533,7 @@ def Embeddings(
         Transformation block to apply after the embeddings lookup, by default None
     aggregation: Optional[TabularAggregationType], optional
         Transformation block to apply for aggregating the inputs, by default None
-    block_name: str, optional
+    block_name: Optional[float, Dict[str, float]] = 0.0
```

Comment:
missing the name and description of the new parameter here `l2_batch_regularization_factor`.